src/app.ts

```diff
@@ -100,12 +101,23 @@ app.all("/", async (req, res) => {
         url: req.url,
       });
 
+      const configPath = process.env.HASURA_DDN_PLUGIN_CONFIG_PATH;
+
+      const configInput = configPath
+        ? require(`${configPath}/configuration.json`)
+        : undefined;
```

Comment:
We are reading and parsing the config on each request. Can we do it once when the server starts or is this fine?

Reply:
Ahh good point, let's switch it to parse once on startup, that way the pod will blow up immediately if it's wrong, rather than waiting for first request.

Reply:
Done!